### PR TITLE
[stable/insights-agent] Remove memory limits from Agent

### PR DESCRIPTION
**Pull request by makoscafee**
**Why This PR?**
_a short description of why this PR is needed_

Fixes #

**Changes**
Changes proposed in this pull request:

*
*

**Checklist:**

* [x] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

### CODE REVIEW

**Comment by rbren** [comment]
@makoscafee can you try adding an Insights exemption for memory limits? https://insights.docs.fairwinds.com/configure/ci/configuration/#fairwinds-insights-yaml

File: stable/insights-agent/values.yaml [hunk 1]
```diff
@@ -561,7 +544,6 @@ falcosecurity:
   resources:
     requests:
       cpu: 100m
-      memory: 256Mi
```
Comment (rbren) [inline stable/insights-agent/values.yaml:564]: This one removed requests (looks like order is reversed)

I think we should remove CPU limits as well.

File: stable/insights-agent/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart to run the Fairwinds Insights agent
 name: insights-agent
-version: 2.2.4
+version: 2.2.5
```
Comment (rbren) [inline stable/insights-agent/Chart.yaml]: Let's release this as a new patch version

File: stable/insights-agent/values.yaml (hunk 1 above)
Comment (makoscafee) [reply stable/insights-agent/values.yaml:564]: CPU limit for all? or just falco?

Comment (rbren) [reply stable/insights-agent/values.yaml:564]: For all

Comment (makoscafee) [reply stable/insights-agent/values.yaml:564]: done

**Review by rbren — APPROVED** [review]

**Review by rbren — COMMENTED** [review]
You'll just need to bump the minor version

**Pull request merged**